www/js/directives/sf_resize.js

```diff
@@ -0,0 +1,21 @@
+app.directive('sfResize', function ($window) {
+  return function (scope, element) {
+    scope.getWinHeight = function() {
+      return $window.innerHeight;
+    }
+
+    var setNavHeight = function(newHeight) {
```

Comment:
I don't think this needs to be a separate function given that it's just one line and used only once.